packages/static-hosting/lib/static-hosting.ts

```diff
@@ -7,6 +7,7 @@ import { CloudFrontTarget } from 'aws-cdk-lib/aws-route53-targets';
 import { User, Group, Policy, PolicyStatement, Effect } from 'aws-cdk-lib/aws-iam';
 
 export interface StaticHostingProps {
+    stackExportPrefix?: string, 
```

Comment:
This is a little nit-picky but I'd probably call this `exportPrefix` as this is a construct not a stack.

Reply:
This is a good call I would change that.